schema/can.proto

```diff
@@ -58,12 +58,13 @@ message CanMsg {
     LIGHTS_FRONT = 5;
     LIGHTS_REAR = 6;
     MOTOR_CONTROLLER = 7;
-    DRIVER_CONTROLS = 8;
-    DRIVER_DISPLAY = 9;
```

Comment:
Since you're taking over the old `DRIVER_DISPLAY` Device ID, can you make sure the other messages that have a `target` annotated are updated? If they're no longer relevant, then remove it, otherwise, update the name.

Reply:
good catch, fixed it